controllers/volumereplicationgroup_controller.go

```diff
@@ -774,6 +774,12 @@ func (v *VRGInstance) processForDeletion() ctrl.Result {
 
 	defer v.log.Info("Exiting processing VolumeReplicationGroup")
 
+	if err := v.disownPVCs(); err != nil {
+		v.log.Info("Disowning PVCs failed", "error", err)
+
+		return ctrl.Result{Requeue: true}
+	}
```

Comment:
This is correct only for the primary vrg when we disable dr, no?

In relocate if we do the final sync after removing the application, we should not disown the pvc - maybe this never reach processForDeletion?

Reply:
We only disown on VRG deletion.  Otherwise, this code does not execute.